.github/workflows/cli_ci.yml

```diff
@@ -94,4 +94,30 @@ jobs:
           ls -lh maestro-k
           file maestro-k
           echo "CLI version:"
-          ./maestro-k --version 
\ No newline at end of file
+          ./maestro-k --version
+
+  lint:
```

Comment:
I spent a lot of time getting the go longing to work. Can we make sure correct version is being used?